src/parser/priest/holy/modules/talents/75/PrayerCircle.tsx

```diff
@@ -0,0 +1,84 @@
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS';
+import React from 'react';
+import Events, { BeginCastEvent, CastEvent } from 'parser/core/Events';
+import Statistic from 'interface/statistics/Statistic';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import AbilityTracker from 'parser/priest/holy/modules/core/AbilityTracker';
+import Abilities from 'parser/priest/holy/modules/Abilities';
+
+const PR_BUFF_DURATION = 8000;
+
+/*
+  Using Circle of Healing reduces the cast time of your Prayer of Healing by 25% for 8 sec.
+*/
+class PrayerCircle extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    abilities: Abilities,
+  };
+  protected abilityTracker!: AbilityTracker;
+  protected abilities!: Abilities;
+
+  lastCohCastAt: number = 0;
+  lastPohCastStart: {
+    timestamp: number,
+    isBuffed: boolean,
+    expectedCooldown: number,
+  } = { timestamp: 0, isBuffed: false, expectedCooldown: 0 };
+  buffedCohCasts = 0;
+
+  get unbuffedCohCasts() {
+    return (this.abilityTracker.getAbility(SPELLS.PRAYER_OF_HEALING.id).casts || 0) - this.buffedCohCasts;
+  }
+
+  constructor(options: any) {
+    super(options);
+    this.active = true; // this.selectedCombatant.hasTalent(SPELLS.PRAYER_CIRCLE_TALENT.id);
```

Comment:
Not sure if you wanted to leave this in for testing; commenting in case it was accidental

Reply:
Removed, thanks for catching that!